### DISCUSSION TITLE
Is there a limit on the number of items that can be viewed as console.log ?

### DISCUSSION THREAD

**Discussion by Luna-omni** [vitest-dev/vitest#4495] [Q&A]
Hello, and thank you for your assistance. I've been using Vitest well.

I noticed a [discussion](https://github.com/vitest-dev/vitest/pull/402 
) related to console.log items in Vitest on GitHub. From what I gathered, there seemed to be an improvement to remove the limit on the number of items shown in console.log. However, when I tested it, it appears that only about 100 items are displayed.

Is there a way to view the entire log without restricting the number of items, perhaps using an alternative method to console.log?

**Comment by sheremet-va** [comment]
This is just how Node.js console works. You can use `console.dir(obj, { maxArrayLength: 3000 })` to show more data.
